resim/examples/sim_runner/test_run.sh

```diff
@@ -5,20 +5,27 @@ set -e
 
 SIM_RUNNER_DIR=resim/examples/sim_runner
 
-# Set up the experience in local folders in a volume which the dev container
-# mounts at /root/.
+# Set up the experience inputs and outputs in docker volumes
+docker container create \
```

Comment:
https://docs.docker.com/engine/reference/commandline/volume_create/

Reply:
Actually you need a container anyway to operate against the volumes so ignore that :)